attributes/repo.rb

```diff
@@ -4,3 +4,9 @@
 default['rundeck_server']['yum']['baseurl']     = 'http://dl.bintray.com/rundeck/rundeck-rpm/'
 default['rundeck_server']['yum']['gpgkey']      = 'http://rundeck.org/keys/BUILD-GPG-KEY-Rundeck.org.key'
 default['rundeck_server']['yum']['action']      = :create
+
+default['rundeck_server']['apt']['uri'] = 'http://dl.bintray.com/rundeck/rundeck-deb/'
+default['rundeck_server']['apt']['components'] = %w(/)
+default['rundeck_server']['apt']['key'] = 'http://rundeck.org/keys/BUILD-GPG-KEY-Rundeck.org.key'
+default['rundeck_server']['apt']['options'] = '--force-yes'
```

Comment:
`apt-get` was reporting issue when using `-y install` which is default options for `apt_package` and adding `--force-yes` was a proposed solution.

However, package can be installed if i execute `apt-get install rundeck` on shell without any issue. To be honest, i did not have time to investigate it further.

I will give it another try, if there is no further issue with the PR. Will remove the extra commit.

Appreciate your feedback.